src/AbstractUri.php

```diff
@@ -362,33 +356,76 @@ protected static function urlEncodeMatch(array $matches): string
      *
      * @return string|null
      */
-    protected function formatHost($host)
+    protected function formatHost(string $host = null)
```

Comment:
This is a BC break unless you require PHP 7.2.

Reply:
I'll revert this change